src/main/kotlin/org/malachite/estella/people/domain/UserRepository.kt

```diff
@@ -8,5 +8,5 @@ interface UserRepository {
     fun findByMail(mail:String): Optional<User>
     fun findById(id: Int): Optional<User>
     fun save(user: User): User
-    fun deleteById(id: Int)
+    fun deleteById(id: Int): User
```

Comment:
Nie jest w końcu tak, że usuwania nic nie zwracają?

Reply:
Mogą zwracać nie muszą a jak na delecie też chcemy przechwycić no such element exception to musimy mieć zwracany taki sam typ (po zamianie na generyk)

Reply:
Znaczy teoretycznie pytanie, czy chcemy coś takiego przechwytywać. Z frontu raczej nie uda Ci się usunąć czegoś, czego nie ma. Pytanie, czy usunięcie czegoś, czego nie ma nas boli?

Reply:
Znaczy wydaje mi się że fajnie żeby operacja delete zwracała 200 zawsze, a nie że jak będziemy usuwali czegoś czego nie ma to dostaniemy 500 bo będzie wyjątek którego nie  przechwycilismy XDD

Reply:
Znaczy tak, pytanie tylko czy delete na czymś, czego nie ma zwraca 500?

Reply:
Znaczy sugerowałem się tym że w którymś innym kontrolerze jest przechwytywanie wyjątków przy metodzie delete więc obstawiałem że to ty robiłeś więc musiało go być dobre XDD

Reply:
Ogólnie powinieneś sprawdzić właśnie dlatego, że tak zrobiłem XDD ale fakt, gdzieś na stacku pisali, że rzuca jakiś exception, co prawda nie wiem, czy NoSuchElement, ale to już pewnie w testach wyjdzie

Reply:
Dobra sprawdzę i zrobię test na to

Reply:
Dobra po zastanowieniu i tak przy usuwaniu wcześniej będziemy sprawdzać czy mamy prawo do takiej operacji więc nie powinnien się zdarzyć taki wyjątek